hivemq-edge/src/main/java/com/hivemq/configuration/entity/adapter/SouthboundMappingEntity.java

```diff
@@ -125,7 +124,7 @@ public void validate(final @NotNull List<ValidationEvent> validationEvents) {
     }
 
     @Override
-    public boolean equals(final Object o) {
+    public boolean equals(final @Nullable Object o) {
         if (this == o) return true;
```

Comment:
missing curly brackets {}

Reply:
the hashCode method makes inconsistent use of getter methods and attributes.

Reply:
Oh, no, that's not my bad. I tend not to touch other code as this PR is already large.